test/test_images/hellovolume/hellovolume.go

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Do you need to update the upload test images shell script for the new target?

Reply:
The script picks up new directories automatically